charts/gateway/README.md

```diff
@@ -1157,6 +1158,51 @@ The full default is this
     com.l7tech.server.clusterStaleNodeCleanupTimeoutSeconds=86400
     # Additional properties go here
 ```
+### DISKLESS Configuraton
+DISKLESS_CONFIG is a flag to pass gateway sensitive data to be mounted to the container gateway file system or passed as environment variables.
+By Default, true, environment variables are used to configure Gateway.
+
+When DISKLESS_CONFIG is false , Gateway will be configured from node.properties
+
+#### node.properties
+Note: 
+- Database configuration should be same as mentioned in node.properties
+```
+node.cluster.pass=newpassword
+admin.user=admin
+admin.pass=newpassword
+node.db.config.main.host=myDBHost.com
```

Comment:
i thought of a case when reading the comment on values.yaml

`jdbcURL: jdbc:mysql://<host>:<port>/<database> | jdbc:mysql://<host>:<port>,<host>:<port>/<database>,...`

this means we need to suppose multiple host:port combo.

Reply:
Added to our spec:

Customers that need to specify multiple db hosts:ports combination would need to look into haproxy or proxysql to expose only 1 host:port info

https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-gateway/11-1/install-configure-upgrade/connecting-to-percona-xtradb-cluster.html